src/pybind/mgr/dashboard/health.html

```diff
@@ -81,15 +81,33 @@
             };
 
             rivets.formatters.pg_status_style = function(pg_status) {
+                var warning = false;
+                var error = false;
+
                 $.each(pg_status, function(state, count) {
-                    if (state == "active+clean"
-                     || state == "active+clean+scrubbing"
-                     || state == "active+clean+scrubbing+deep") {
-                        return "color: #00bb00";
-                    } else {
-                        return "color: #FFC200";
+                    if (state.includes("inconsistent") ||
+                        state.includes("incomplete") ||
+                        !state.includes("active")
+                    ) {
+                        error = true;
+                    }
+
+                    if (state != "active+clean"
+                     && state != "active+clean+scrubbing"
```

Comment:
I think we shouldn't be indicating warnings for scrubbing states

Reply:
I think you read the code wrongly. It excludes scrubbing from the state where a PG would be in warning state

Reply:
Oops!  My mistake.  Making another coffee now.